grafana-plugin/playwright.config.ts

```diff
@@ -53,23 +54,33 @@ const config: PlaywrightTestConfig = {
 
   /* Configure projects for major browsers */
   projects: [
+    {
+      name: 'setup',
+      testMatch: /globalSetup\.ts/,
+    },
```

Comment:
The [Playwright docs](https://playwright.dev/docs/test-global-setup-teardown) mentioned that defining the global setup as a "project" step here, it allows us to capture traces/videos on failures during this phase (global setup down via the `globalSetup` configuration attribute does not have this benefit).